src/hotspot/cpu/aarch64/templateTable_aarch64.cpp

```diff
@@ -2357,32 +2366,111 @@ void TemplateTable::load_resolved_field_entry(Register obj,
   }
 }
 
-// The Rcache and index registers must be set before call
-// n.b unlike x86 cache already includes the index offset
-void TemplateTable::load_field_cp_cache_entry(Register obj,
-                                              Register cache,
-                                              Register index,
-                                              Register off,
-                                              Register flags,
-                                              bool is_static = false) {
-  assert_different_registers(cache, index, flags, off);
+void TemplateTable::load_resolved_method_entry_common(Register cache,
```

Comment:
When I saw the `common` suffix I thought at first that this might be used in addition to the variants with the other suffixes rather than as an alternative. Since this is 'common' to only to 2 out of the 5 different cases perhaps it could be named `load_resolved_method_entry_special_or_static`. That would make it clear that each invoke type has a corresponding `load_resolved_method_entry_<invoketype>` variant.

Reply:
Yes, it's a long name but it clarifies the intended purpose consistently with the names for the other cases. I think it is best to stick with it.